modules/app/components/SystemStatsSidebar.tsx

```diff
@@ -38,6 +39,42 @@ type StatField =
   | 'debt ceiling'
   | 'system surplus';
 
+const PollingContractsModal = () => {
+  const [overlayOpen, setOverlayOpen] = useState(false);
+
+  return (
+    <>
+      <Flex onClick={() => setOverlayOpen(true)} sx={{ cursor: 'pointer', ml: 1 }}>
+        <Icon name="info" color="primary" />
+      </Flex>
+      {overlayOpen && (
+        <DialogOverlay isOpen={overlayOpen} onDismiss={() => setOverlayOpen(false)}>
+          <DialogContent ariaLabel="Polling contract versions info">
+            <BoxWithClose close={() => setOverlayOpen(false)}>
+              <Flex
+                sx={{
+                  flexDirection: 'column',
+                  justifyContent: 'center',
+                  alignItems: 'center'
+                }}
+              >
+                <Heading sx={{ mb: 3 }}>Polling contract versions</Heading>
+                <Text sx={{ textAlign: 'center' }}>
+                  - The latest version of the polling contract was deployed to enable batch voting, so users
```

Comment:
Should we put something like V1: <description> and V2: <description> so we know which description corresponds to both? Since they both open the same modal?

Reply:
See my comment below on the other modal text regarding `p` vs `span` for this content.